src/main/java/bro.java

```diff
@@ -1,13 +1,119 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class bro {
+
+    private static ArrayList<String> storer = new ArrayList<>();
+    private static ArrayList<String> mark_tracker = new ArrayList<>();
+
+    public static void level0() {
+        System.out.println("Hello! I'm bro");
+        System.out.println("What can I do for you?");
+        System.out.println("Bye. Hope to see you again soon!");
```

Comment:
This line should not be here? The chatbot prints Bye almost instantly

Reply:
Bye should not be in level 0